Impossible-Agent-Suite/DangerousActivities/src/agent.spec.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+} from "forta-agent";
+import { provideHandleTransaction } from "./agent";
+import { createAddress, TestTransactionEvent } from "forta-agent-tools";
+import utils from "./utils";
+
+const toAddresses = (list: string[]) => list.map(x => createAddress(x));
+
+const createFinding = (impossible: string[], dangerous: string[]) => Finding.fromObject({
+  name: "Impossible Finance interaction monitor",
+  description: "Dangerous Interaction detected",
```

Comment:
If you follow my suggestion for the change of `description` in `agent.ts` then this would have to be changed too.